server/models/Transaction.ts

```diff
@@ -1085,9 +1085,14 @@ Transaction.createHostFeeTransactions = async (transaction, host, data) => {
 Transaction.getPaymentProcessorFeeVendor = memoize(
   async (service: PAYMENT_METHOD_SERVICE | PayoutMethodTypes | 'OTHER'): Promise<Collective> => {
     const vendorSlugs = {
+      // Stripe
       [PAYMENT_METHOD_SERVICE.STRIPE]: 'stripe-payment-processor-vendor',
+      // Paypal
       [PAYMENT_METHOD_SERVICE.PAYPAL]: 'paypal-payment-processor-vendor',
-      [PayoutMethodTypes.BANK_ACCOUNT]: 'wise-payment-processor-vendor', // TODO: We should differentiate depending on manual vs. automatic
+      [PayoutMethodTypes.PAYPAL]: 'paypal-payment-processor-vendor',
+      // Wise
+      [PayoutMethodTypes.BANK_ACCOUNT]: 'wise-payment-processor-vendor',
+      // Manual
```

Comment:
Made another update here: Payout/Payment method services are not formatted the same way (lowercase vs uppercase), we would have missed the vendor for PayPal expenses without this change